src/app/testimonials/page.tsx

```
@@ -1,93 +1,77 @@
+  }
+
+  if (error) {
+    return (
+      <div className="min-h-screen bg-gradient-hero text-white flex-center">
+        <div className="text-center p-8">
+          <h2 className="text-2xl font-bold mb-4">Error Loading Testimonials</h2>
+          <p className="text-gray-300 mb-4">{(error as Error).message}</p>
+          <button 
+            onClick={() => window.location.reload()}
+            className="px-6 py-3 bg-cyan-600 hover:bg-cyan-700 rounded-lg transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)